docs/reference/objects/blog_post.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Blog Post
+parent: Objects
+---
+
+When using a `blog_post` object you have access to the following attributes.
+
```

Comment:
@ianmooney I've updated this, could you take a quick look at the type one because the documented behaviour is a bit odd and a sense check would be great.